File: app/src/main/java/com/bopr/android/smailer/ui/TextBlacklistActivity.java
package com.bopr.android.smailer.ui;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

/**
 * Number blacklist activity.
 *
 * @author Boris Pronin (<a href="mailto:boprsoft.dev@gmail.com">boprsoft.dev@gmail.com</a>)
 */
public class TextBlacklistActivity extends AppActivity {

    @NonNull
    @Override
    protected Fragment createFragment() {
        return new TextBlacklistFragment();
    }

}

File: app/src/main/kotlin/com/bopr/android/smailer/ui/TextBlacklistActivity.kt
package com.bopr.android.smailer.ui

import androidx.fragment.app.Fragment

/**
 * Number blacklist activity.
 *
 * @author Boris Pronin ([boprsoft.dev@gmail.com](mailto:boprsoft.dev@gmail.com))
 */
class TextBlacklistActivity : AppActivity() {

    override fun createFragment(): Fragment {
        return TextBlacklistFragment()
    }
}